go.sum

```diff
@@ -81,8 +81,9 @@ github.com/cespare/xxhash/v2 v2.2.0/go.mod h1:VGX0DQ3Q6kWi7AoAeZDth3/j3BFtOZR5XL
 github.com/checkpoint-restore/go-criu/v5 v5.3.0/go.mod h1:E/eQpaFtUKGOOSEBZgmKAcn+zUUwWxqcaKZlF54wK8E=
 github.com/cilium/ebpf v0.7.0/go.mod h1:/oI2+1shJiTGAMgl6/RgJr36Eo1jzrRcAWbcXO2usCA=
 github.com/client9/misspell v0.3.4/go.mod h1:qj6jICC3Q7zFZvVWo7KLAzC3yx5G7kyvSDkc90ppPyw=
-github.com/cloudflare/circl v1.3.3 h1:fE/Qz0QdIGqeWfnwq0RE0R7MI51s0M2E4Ga9kq5AEMs=
 github.com/cloudflare/circl v1.3.3/go.mod h1:5XYMA4rFBvNIrhs50XuiBJ15vF2pZn4nnUKZrLbUZFA=
```

Comment:
I'm not sure if this leftover in `go.sum` still means an indirect dependency somewhere?

Reply:
Yes.

I looked through the source code. It looks like `cloudflare/circl` is an indirect dependency in any case, and we do not use functions or methods from the library directly.

Most probably it's a dependency of more than one of our dependencies and some upgraded already and some didn't.

Reply:
Well anyhow, it's better to remove a direct dependency on the vulnerable version even if it doesn't completely purge it from the graph.